src/lib/constants.ts

```diff
@@ -0,0 +1,47 @@
+import { checkString } from './validation';
+
+const bootMountPointFromEnv = checkString(process.env.BOOT_MOUNTPOINT);
+const rootMountPoint = checkString(process.env.ROOT_MOUNTPOINT) || '/mnt/root';
+
+const supervisorNetworkInterface = 'supervisor0';
+
+export = {
+	rootMountPoint,
+	databasePath: checkString(process.env.DATABASE_PATH) || '/data/database.sqlite',
+	dockerSocket: process.env.DOCKER_SOCKET || '/var/run/docker.sock',
+	supervisorImage: checkString(process.env.SUPERVISOR_IMAGE) || 'resin/rpi-supervisor',
+	ledFile: checkString(process.env.LED_FILE) || '/sys/class/leds/led0/brightness',
+	// Only used in development
+	forceSecret: {
+		api: checkString(process.env.RESIN_SUPERVISOR_SECRET) || null,
+		logsChannel: checkString(process.env.RESIN_SUPERVISOR_LOGS_CHANNEL) || null,
+	},
+	vpnStatusPath:
+		checkString(process.env.VPN_STATUS_PATH) || `${rootMountPoint}/run/openvpn/vpn_status`,
+	hostOSVersionPath:
+		checkString(process.env.HOST_OS_VERSION_PATH) || `${rootMountPoint}/etc/os-release`,
+	privateAppEnvVars: [
+		'RESIN_SUPERVISOR_API_KEY',
+		'RESIN_API_KEY',
+	],
+	dataPath: checkString(process.env.RESIN_DATA_PATH) || '/resin-data',
+	bootMountPointFromEnv,
+	bootMountPoint: bootMountPointFromEnv || '/boot',
+	configJsonPathOnHost: checkString(process.env.CONFIG_JSON_PATH),
+	proxyvisorHookReceiver:
+		checkString(process.env.RESIN_PROXYVISOR_HOOK_RECEIVER) || 'http://0.0.0.0:1337',
+	apiEndpointFromEnv: checkString(process.env.API_ENDPOINT),
+	configJsonNonAtomicPath: '/boot/config.json',
+	defaultPubnubSubscribeKey: process.env.DEFAULT_PUBNUB_SUBSCRIBE_KEY,
+	defaultPubnubPublishKey: process.env.DEFAULT_PUBNUB_PUBLISH_KEY,
+	defaultMixpanelToken: process.env.DEFAULT_MIXPANEL_TOKEN,
+	supervisorNetworkInterface: supervisorNetworkInterface,
+	allowedInterfaces: [ 'resin-vpn', 'tun0', 'docker0', 'lo', supervisorNetworkInterface ],
```

Comment:
I just realized that this is outdated,we should probably think about what interfaces the supervisor listens on... I think it might only need to be resin-vpn, lo, and the supervisorNetworkInterface. But it might be better left for a separate PR.

Reply:
Yeah I'll save it for another PR, making an issue now.

Reply:
https://github.com/resin-io/resin-supervisor/issues/639